api/nodes/serializers.py

```diff
@@ -336,9 +338,36 @@ def get_logs_count(self, obj):
 
     def get_node_count(self, obj):
         auth = get_user_auth(self.context['request'])
-        return len([node_relation
-                    for node_relation in obj.node_relations.filter(child__is_deleted=False, is_node_link=False)
-                    if node_relation.child.can_view(auth)])
+        user_id = getattr(auth.user, 'id', None)
+        with connection.cursor() as cursor:
```

Comment:
@chrisseto You said there were tests that initially did not pass and they did pass after that, correct? In running the full suite I found that occasionally `auth` will be `None` and you would get a `AttributeError` which is why lino 341 was added.